.claude/commands/analyze.md

```
@@ -0,0 +1,200 @@
+### 3단계: 분석 시 스타일 적용 원칙
+✅ **절대 원칙**: 프로젝트의 기존 코드 스타일을 100% 따라감  
+✅ 일관성 > 베스트 프랙티스  
+✅ 팀 컨벤션 > 개인 취향  
+✅ 새로운 스타일 제안 금지 (명시적 요청 시에만)
+
+---
+
+## 핵심 원칙
+- ✅ 철저한 분석과 계획 수립
+- ❌ 직접적인 코드 수정 금지
+- ✅ 문제점과 개선 방향 제시
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)